content/en/docs/collector/custom-collector.md

```diff
@@ -20,29 +20,67 @@ builder][ocb] (or `ocb` for short) to assist people in assembling their own
 distribution, making it easy to build a distribution that includes their custom
 components along with components that are publicly available.
 
-As part of the process the `builder` will generate the Collector's source code,
+As part of the process the `ocb` will generate the Collector's source code,
 which you can use to help build and debug your own custom components, so let's
 get started.
 
 ## Step 1 - Install the builder
 
 The `ocb` binary is available as a downloadable asset from [OpenTelemetry
-Collector releases][releases]. You will find the list of assets at the bottom of
-the page. Assets are named based on OS and chipset, so download the one that
-fits your configuration.
+Collector releases][releases]. You will find a list of assets named based on OS
+and chipset, so download the one that fits your configuration:
 
-The binary has a pretty long name, so you can simply rename it to `ocb`; and if
-you are running Linux or macOS, you will also need to provide execution
-permissions for the binary.
+{{< tabpane text=true >}}
```

Comment:
Nit, and maybe not something to be fixed on this PR, but on a 16 inch mac, the tabs are too big, and take 2 lines.

<img width="918" alt="Screenshot 2024-03-11 at 17 08 51" src="https://github.com/open-telemetry/opentelemetry.io/assets/9347/d506ebd1-d766-4150-bef2-2061e5965a10">